api/napi/src/types/image_data.rs

```diff
@@ -9,6 +9,10 @@ use i_slint_core::{
 };
 use napi::bindgen_prelude::{Buffer, External};
 
+// This is needed for typdoc check JsImageData::image
+pub type ImageData = Image;
+
+/// An image data type that can be displayed by the Image element
```

Comment:
Please elaborate on how this type can be used, preferably with an example. For example to introspect the size of an image loaded by Slint, and by integrating image data produces by a "third party package" into slint.

It might be worth porting the image filter example to JavaScript.